docs/content/SplitPageLayout.mdx

```diff
@@ -227,9 +300,19 @@ description: Provides structure for a split page layout, including independent s
 <PropsTable>
   <PropsTableRow
     name="position"
-    type={`ResponsiveValue<
-  'start' | 'end'
->`}
+    type={`| 'start'
```

Comment:
Non-blocking: This is much clearer to the docs reader than `ResponsiveValue<'start' | 'end'>`, but it will be annoying to update when/if we change our breakpoints.

Reply:
Agreed. This is a temporary solution until we can think of a better way to document responsive values. Open to ideas!